modules/apps/object/object-api/src/main/java/com/liferay/object/exception/ObjectDefinitionExternalReferenceCodeException.java

```diff
@@ -22,24 +22,46 @@ public ForbiddenUnmodifiableSystemObjectDefinitionExternalReferenceCode(
 			String externalReferenceCode) {
 
 			super(
+				StringBundler.concat(
+					"Forbidden unmodifiable system object definition external ",
+					"reference code ", externalReferenceCode),
 				StringBundler.concat(
 					"Forbidden unmodifiable system object definition external ",
 					"reference code ", externalReferenceCode));
 		}
 
 	}
 
+	public static class MustBeLessThan75Characters
+		extends ObjectDefinitionExternalReferenceCodeException {
+
+		public MustBeLessThan75Characters() {
+			super(
+				"ERC must be less than 75 characters",
+				"only-75-characters-are-allowed");
+		}
+
+	}
+
 	public static class MustNotStartWithPrefix
 		extends ObjectDefinitionExternalReferenceCodeException {
 
 		public MustNotStartWithPrefix() {
-			super("The prefix L_ is reserved for Liferay");
+			super(
+				"The prefix L_ is reserved for Liferay",
+				"The prefix L_ is reserved for Liferay");
```

Comment:
We need to create the language key, something like this:
`the-prefix-x-is-reserved-for-liferay=The prefix {0} is reserved for Liferay`

And follow the same pattern the comment above, please

Reply:
One more thing: I observed that we use "L_" in several places in the code. We can add a new constant in `ObjectDefinitionConstants.java`, something like this:
`public static final String PREFIX_SYSTEM_OBJECT_DEFINITION_EXTERNAL_REFERENCE_CODE = "L_"`

and replace in all place that use the "L_" hard code.